app/templates/components/post-header.hbs

```diff
@@ -1,2 +1,2 @@
 <div class="post-icon"></div>
-{{post-title post=post}}
+{{post-title post=post saveTitle=saveTitle}}
```

Comment:
Generally, the naming that makes sense to me would be something like

```Handlebars
{{post-title save='savePostTitle'}}
```

The reasoning here is
* The post title component has one job, one action, so there's no need for the name to say specifically what the action does. It should be clear from the component itself.

Past that component, though, the name of the action that handles it should be more specific.

I'm even thinking of going a step further. The `post-title` component isn't really saving anymore. It's actually, `applying` or `cancelling` the `edit`. so maybe the action should be `applyEdit`. Maybe, even, the component should be named `editable-post-title` to make it even more self-commenting.